poc/vdaf_poc/field.py

```diff
@@ -272,6 +264,103 @@ def poly_eval(field: type[F], p: list[F], eval_at: F) -> F:
 
 def poly_interp(field: type[F], xs: list[F], ys: list[F]) -> list[F]:
     """Compute the Lagrange interpolation polynomial for the given points."""
-    R = PolynomialRing(field.gf, 'x')
-    p = R.lagrange_polynomial([(x.val, y.val) for (x, y) in zip(xs, ys)])
-    return poly_strip(field, list(map(field, p.coefficients())))
```

Comment:
I would add a historical note here that we previously invoked the reference code with sage, which gave us a faster implementation of this algorithm. The note should give justification for this (dropping sage makes the code more portable).